src/ducks/transactions/Transactions.styl

```diff
@@ -32,6 +32,9 @@
     .bnk-op-caticon
         cursor pointer
 
+    .bnk-op-label
+        max-width 100%
```

Comment:
I think this should be done automatically on `media__body`

Reply:
Can you add a TODO here ?

Reply:
I don't understand why this is necessary ? Has the ListItemText a problem with Media Bd ?